powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java

```diff
@@ -83,7 +83,8 @@ public ObjectResponse<VerifySignatureResponse> verifySignature(@RequestBody Obje
     public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createPersonalizedOfflineSignaturePayload(@RequestBody ObjectRequest<CreatePersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
         logger.info("CreatePersonalizedOfflineSignaturePayloadRequest received: {}", request);
```

Comment:
Doesn't logging of the request object cause same issue?
Maybe we could log only the activation ID on info level.

Reply:
yes, do not log the whole object on info level. And if so apply our logging guideline. action: {}, state: {},...